### PR TITLE
Fix various sanity errors in plugins

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Fix some errors found with ansible/ansible#71734.

##### ISSUE TYPE
- Bugfix Pull Request
- Docs Pull Request

##### COMPONENT NAME
plugins/become/pfexec.py
plugins/callback/cgroup_memory_recap.py
plugins/callback/counter_enabled.py
plugins/callback/diy.py
plugins/callback/hipchat.py
plugins/callback/logdna.py
plugins/callback/mail.py
plugins/callback/null.py
plugins/callback/selective.py
plugins/callback/slack.py
plugins/callback/stderr.py
plugins/callback/syslog_json.py
plugins/inventory/gitlab_runners.py
plugins/lookup/chef_databag.py
plugins/lookup/consul_kv.py
plugins/lookup/credstash.py
plugins/lookup/cyberarkpassword.py
plugins/lookup/dig.py
plugins/lookup/dsv.py
plugins/lookup/etcd3.py
plugins/lookup/filetree.py
plugins/lookup/gcp_storage_file.py
plugins/lookup/hashi_vault.py
plugins/lookup/hiera.py
plugins/lookup/keyring.py
plugins/lookup/lastpass.py
plugins/lookup/lmdb_kv.py
plugins/lookup/nios.py
plugins/lookup/nios_next_ip.py
plugins/lookup/nios_next_network.py
plugins/lookup/onepassword.py
plugins/lookup/onepassword_raw.py
plugins/lookup/passwordstore.py
plugins/lookup/redis.py
plugins/lookup/shelvefile.py
plugins/lookup/tss.py

### CODE REVIEW

File: plugins/lookup/chef_databag.py [hunk 1]
```diff
@@ -35,6 +35,8 @@
   _raw:
     description:
       - The value from the databag
```
Comment (Andersson007) [inline plugins/lookup/chef_databag.py:37]:
```suggestion
      - The value from the databag.
```
If this is ubiquitous there, don't fix

File: plugins/lookup/credstash.py [hunk 2]
```diff
@@ -79,6 +79,7 @@
   _raw:
     description:
       - value(s) stored in Credstash
```
Comment (Andersson007) [inline plugins/lookup/credstash.py:81]:
```suggestion
      - Value(s) stored in Credstash.
```
the same, feel free to ignore my comment if all stuff is like this there

File: plugins/lookup/cyberarkpassword.py [hunk 3]
```diff
@@ -30,7 +30,7 @@
         default: 'password'
       _extra:
         description: for extra_params values please check parameters for clipasswordsdk in CyberArk's "Credential Provider and ASCP Implementation Guide"
-    note:
+    notes:
       - For Ansible on windows, please change the -parameters (-p, -d, and -o) to /parameters (/p, /d, and /o) and change the location of CLIPasswordSDK.exe
```
Comment (Andersson007) [inline plugins/lookup/cyberarkpassword.py:34]:
```suggestion
      - For Ansible on Windows, please change the -parameters (-p, -d, and -o) to /parameters (/p, /d, and /o) and change the location of CLIPasswordSDK.exe.
```

File: plugins/lookup/dig.py [hunk 4]
```diff
@@ -79,6 +79,8 @@
     description:
       - list of composed strings or dictionaries with key and value
```
Comment (Andersson007) [inline plugins/lookup/dig.py:80]:
```suggestion
      - List of composed strings or dictionaries with key and value
```

File: plugins/lookup/filetree.py [hunk 5]
```diff
@@ -54,41 +54,59 @@
...
           - not returned when C(item.state) is set to C(directory).
+          type: path
         root:
           description: allows filtering by original location.
+          type: path
         path:
           description: contains the relative path to root.
+          type: path
         mode:
           description: The permissions the resulting file or directory.
```
Comment (Andersson007) [inline plugins/lookup/filetree.py:72]:
```suggestion
          - Full path to file.
          - Not returned when I(item.state) is set to C(directory).
          type: path
        root:
          description: Allows filtering by original location.
          type: path
        path:
          description: Contains the relative path to root.
          type: path
        mode:
          description: The permissions the resulting file or directory.
```

File: plugins/lookup/filetree.py [hunk 6]
```diff
@@ -54,41 +54,59 @@
 RETURN = r"""
   _raw:
     description: list of dictionaries with file information
```
Comment (Andersson007) [inline plugins/lookup/filetree.py:56]:
```suggestion
    description: List of dictionaries with file information.
```

File: plugins/lookup/filetree.py [hunk 7]
```diff
@@ -54,41 +54,59 @@
...
+          type: int
         size:
           description: Size of the target.
+          type: int
         mtime:
           description: Time of last modification.
+          type: int
         ctime:
           description: Time of last metadata update or creation (depends on OS).
+          type: int
```
Comment (Andersson007) [inline plugins/lookup/filetree.py:109]:
```suggestion
          description: Time of last metadata update or creation (depends on OS).
          type: int
```
100% int? Can't this be 'YYYY-MM-DD' or something?

File: plugins/lookup/keyring.py [hunk 8]
```diff
@@ -30,6 +30,8 @@
 RETURN = """
   _raw:
     description: secrets stored
```
Comment (Andersson007) [inline plugins/lookup/keyring.py:32]:
```suggestion
    description: Secrets stored.
```

**Review by Andersson007 — COMMENTED** [review]
@felixfontein feel free to ignore all cosmetic suggestion.. When i started i didn't see that there is a total mess with formatting, etc.

File: plugins/lookup/filetree.py (hunk 7 above)
Comment (felixfontein) [reply plugins/lookup/filetree.py:109]: In the code `ret['ctime'] = st.st_ctime`, and where `st` is the result of https://docs.python.org/3/library/os.html#os.lstat, i.e. an stat_result object (https://docs.python.org/3/library/os.html#os.stat_result). The documentation is unfortunately not very clear.

I've checked typeshed, and interestingly it says float: https://github.com/python/typeshed/blob/master/stdlib/3/os/__init__.pyi#L242

I'll adjust the PR.

**Comment by felixfontein** [comment]
Indeed, there's a huge mess in the plugin docs as well :) I've still commited your suggestions so at least they get fixed.

**Review by Andersson007 — APPROVED** [review]
LGTM

**Comment by felixfontein** [comment]
@Andersson007 thanks a lot for reviewing this!

**Pull request merged**